docs/getting-started.rst

```diff
@@ -0,0 +1,88 @@
+Getting Started
+===============
+On this page, you will find instructions on how to set up your coding environment to contribute to the project.
+
+Prerequisites
+-------------
+In order to contribute to the project, you should have the following prerequisites:
+
+# Install `Git <https://git-scm.com/>`_.
+# Ensure you have Python 3.12 installed or higher. You can check which version you are running by executing the following command in your terminal::
+
+    $ python --version
+
+# Install `Poetry <https://python-poetry.org/>`_ by running one of the following commands in your terminal, depending on your OS:
```

Comment:
I would just direct to Poetry's documentation for installation, it has changed frequently in the past.